### PR TITLE
feat: add start and stop taskmanager in onebox script

### LINKED ISSUE #1047: Add TaskManager in onebox script so that we can run tests with TaskManager
We should add script to start TaskManager in onebox environment and then enable unit tests witch rely on TaskManager.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Resolve https://github.com/4paradigm/OpenMLDB/issues/1047 .

### CODE REVIEW

**Review by dl239 — APPROVED** [review]

File: onebox/start_onebox.sh [hunk 1]
```diff
@@ -88,6 +87,32 @@ test -d recycle_bin3 && rm -rf recycle_bin3
                    --zk_cluster=${ZK_CLUSTER}\
                    --zk_root_path=/onebox > ns3.log 2>&1 &
 
+# start TaskManager if exists
+if [ -d "../java/openmldb-taskmanager/target/" ]
+then
+  # Change to binary directory
+  pushd ../java/openmldb-taskmanager/target/openmldb-taskmanager-binary/bin > /dev/null
```
Comment (aceforeverd) [inline onebox/start_onebox.sh]: relative path not always work when executing script in different directory. instead, simply add
```sh
cd "$(dirname "$0")/.."
```
in line 18
and assume all path below is relative to the project  root

File: onebox/start_onebox.sh [hunk 2]
```diff
@@ -88,6 +87,32 @@ test -d recycle_bin3 && rm -rf recycle_bin3
...
+
+  # Get OS and check
+  OS="$(uname -a | awk '{print $1}' | tr '[:upper:]' '[:lower:]')"
```
Comment (aceforeverd) [inline onebox/start_onebox.sh]: there is '$OSTYPE` provided by bash

Comment (tobegit3hub) [reply onebox/start_onebox.sh]: Okey, we will change to use `$OSTYPE`.

**Review by aceforeverd — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[Implementation of TaskManager](https://trello.com/c/EkxZMO7i/28-implementation-of-taskmanager)

**Pull request merged**